.circleci/config.yml

```diff
@@ -62,29 +62,25 @@ jobs:
           path: test_output/report.xml
       - store_artifacts:
           path: htmlcov
-  run_integration_tests:
```

Comment:
In the parallel run, tests are failing. Switching them to run in a serialized manner.